label_studio/organizations/urls.py

```diff
@@ -3,36 +3,48 @@
 from django.urls import include, path
 from organizations import api, views
 
-app_name = 'organizations'
+app_name = "organizations"
 
 # TODO: there should be only one patterns list based on API (with api/ prefix removed)
 # Page URLs
 _urlpatterns = [
     # get organization page
-    path('', views.organization_people_list, name='organization-index'),
+    path("", views.organization_people_list, name="organization-index"),
 ]
 
 # API URLs
 _api_urlpattens = [
     # organization list viewset
-    path('', api.OrganizationListAPI.as_view(), name='organization-list'),
+    path("", api.OrganizationListAPI.as_view(), name="organization-list"),
     # organization detail viewset
-    path('<int:pk>', api.OrganizationAPI.as_view(), name='organization-detail'),
+    path("<int:pk>", api.OrganizationAPI.as_view(), name="organization-detail"),
     # organization memberships list viewset
-    path('<int:pk>/memberships', api.OrganizationMemberListAPI.as_view(), name='organization-memberships-list'),
     path(
-        '<int:pk>/memberships/<int:user_pk>/',
+        "<int:pk>/memberships",
+        api.OrganizationMemberListAPI.as_view(),
+        name="organization-memberships-list",
+    ),
+    path(
+        "<int:pk>/memberships/<int:user_pk>/",
         api.OrganizationMemberDetailAPI.as_view(),
-        name='organization-membership-detail',
+        name="organization-membership-detail",
     ),
 ]
 # TODO: these urlpatterns should be moved in core/urls with include('organizations.urls')
 urlpatterns = [
-    path('organization/', views.simple_view, name='organization-simple'),
-    path('organization/webhooks', views.simple_view, name='organization-simple-webhooks'),
-    path('people/', include(_urlpatterns)),
-    path('api/organizations/', include((_api_urlpattens, app_name), namespace='api')),
+    path("organization/", views.simple_view, name="organization-simple"),
+    path(
+        "organization/webhooks", views.simple_view, name="organization-simple-webhooks"
+    ),
+    path("people/", include(_urlpatterns)),
+    # TODO: temporary route, remove as needed
+    path("prompt/", views.simple_view, name="prompter"),
```

Comment:
do we really need to add all urls to python routing? I thought it's handled by react 😭

Reply:
there must be a route on the backend, unfortunately. otherwise reloading the page will cause 404